modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -0,0 +1,430 @@
+| `neo4j_dbms_page_cache_hit_ratio_per_minute`
+| _Gauge_
+| AVG
+
+| Page Cache Usage Ratio
+| The percentage of the allocated page cache in use. If this is close to or at 100%, then it is likely that the hit ratio will start dropping, and you should consider increasing the size of your instance so that more memory is available for the page cache.
+| `neo4j_dbms_page_cache_usage_ratio`
+| _Gauge_
+| MIN
+
+| Bolt Connections Running
+| The total number of Bolt connections that are currently executing Cypher transactions and returning results. This is a set of snapshots over time and may appear to spike if workloads are all completed quickly.
+| `neo4j_dbms_bolt_connections_running`
+| _Gauge_
+| MAX
+
+| Bolt Connections Idle
+| The total number of Bolt connections that are connected to the Aura database but not currently executing Cypher or returning results.
+| `neo4j_dbms_bolt_connections_idle`
+| _Gauge_
+| MAX
+
+| Bolt Connections Closed
+| The total number of Bolt connections closed since startup. This includes both properly and abnormally ended connections. This value may drop if background maintenance is performed by Aura.
+| `neo4j_dbms_bolt_connections_closed_total`
+| _Counter_
+| MAX
+
+| Bolt Connections Opened
+| The total number of Bolt connections opened since startup. This includes both successful and failed connections. This value may drop if background maintenance is performed by Aura.
+| `neo4j_dbms_bolt_connections_opened_total`
+| _Counter_
+| MAX
+
+| Garbage Collection Young Generation
+| Shows the total time since startup spent clearing up heap space for short lived objects. Young garbage collections typically complete quickly, and the Aura instance will wait whilst the garbage collector is run. High values indicate that the instance is running low on memory for the workload and you should consider increasing the size of your instance.
+| `neo4j_dbms_vm_gc_time_g1_young_generation_total`
+| _Counter_
+| MAX
+
+| Garbage Collection Old Generation
+| Shows the total time since startup spent clearing up heap space for long lived objects. Old garbage collections can take time to complete, and the Aura instance will wait whilst the garbage collector is run. High values indicate that there are long running processes or queries which could be optimized, or your instance is running low on CPU or memory for the workload and you should consider reviewing these metrics and possibly increasing the size of your instance.
+| `neo4j_dbms_vm_gc_time_g1_old_generation_total`
+| _Counter_
+| MAX
+
+| Replan Events
+| The total number of times Cypher has replanned a query since the server started. If this spikes or is increasing, check that the queries executed are using parameters correctly. This value may drop if background maintenance is performed by Aura.
```

Comment:
```suggestion
| The total number of times Cypher has replanned a query since the server started. 
If this spikes or is increasing, check that the queries executed are using parameters correctly. 
This value may drop if background maintenance is performed by Aura.
```